docs/tutorial/using-multiple-themes-together.md

```diff
@@ -2,4 +2,530 @@
 title: Using Multiple Themes Together
 ---
 
-_This tutorial is coming soon. In the meantime, check out the [introduction to using multiple themes in the docs](/docs/themes/using-multiple-gatsby-themes)._
+## What this tutorial covers
+
+This tutorial covers how to compose multiple themes into a final site using [gatsby-theme-blog](/packages/gatsby-theme-blog/), [gatsby-theme-notes](/packages/gatsby-theme-notes/) and [gatsby-mdx-embed](/packages/@pauliescanlon/gatsby-mdx-embed/) as examples. It will also cover the concept of component shadowing with [Theme-UI](docs/theme-ui/) for styling.
+
+## Prerequisites
+
+This tutorial assumes the following:
+
+- That you have an understanding of [Gatsby fundamentals](/tutorial/#gatsby-fundamentals)
+- An existing knowledge of [Gatsby themes](/docs/themes/what-are-gatsby-themes/)
+
+## Step 1: Create a new site using the hello world starter and change directory to "multiple-themes"
+
+```shell
+gatsby new multiple-themes https://github.com/gatsbyjs/gatsby-starter-hello-world
+```
+
+```shell
+cd multiple-themes
+```
+
+## Step 2: Install and compose the themes
+
+This step composes [gatsby-theme-blog](/packages/gatsby-theme-blog/) and [gatsby-theme-notes](/packages/gatsby-theme-notes/).
+
+Install the themes:
+
+```shell
+npm install gatsby-theme-blog gatsby-theme-notes
+```
+
+Edit `gatsby-config.js` to add the themes to the plugin array and to update the site metadata:
+
+```javascript:title=gatsby-config.js
+module.exports = {
+  siteMetadata: {
+    title: `Your Site Title`,
+    description: `A description for your blazing fast site, using multiple themes!`,
+    author: `Your name`,
+    social: [
+      {
+        name: `Twitter`,
+        url: `https://twitter.com/gatsbyjs`,
+      },
+      {
+        name: `GitHub`,
+        url: `https://github.com/gatsbyjs`,
+      },
+    ],
+  },
+  plugins: [
+    {
+      resolve: `gatsby-theme-blog`,
+      options: {
+        basePath: `/blog`,
+      },
+    },
+    {
+      resolve: `gatsby-theme-notes`,
+      options: {
+        basePath: `/notes`,
+      },
+    },
+  ],
+}
+```
+
+## Step 3: Run the site
+
+Next, run the site and see what you have:
+
+```shell
+cd multiple-themes
+gatsby develop
+```
+
```

Comment:
Wondering if this should be its own step since it's sort of an in between step? Perhaps we could put it as part of the previous section and note that you'll want to run your site periodically to check on your changes.